modules/serverless-logic-upgrade-1.36-finalizing-job-service.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-upgrading-operator-from-1.35-to-1.36
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-upgrade-1.36-finalizing-job-service_{context}"]
+= Finalizing the Job Service upgrade
+
+After the {ServerlessOperatorName} is upgraded to version 1.36.0, a new ReplicaSet for the Job Service component is created automatically. To clean up older resources, you can delete the Job Service ReplicaSets created by the previous 1.35.0 version.
+
+.Procedure
+
+. List all ReplicaSets for the Job Service by running the following command:
++
+[source,terminal]
+----
+$ oc get replicasets -o custom-columns=Name:metadata.name,Image:spec.template.spec.containers[*].image -n <target_namespace>
+----
++
+.Example output
+[source,sql]
+----
+Name                                                Image
+sonataflow-platform-jobs-service-1111111111         registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8:1.35.0
```

Comment:
```suggestion
*sonataflow-platform-jobs-service-1111111111         registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8:1.35.0*
```

Reply:
I have added the *blabla* with hope that this particular line is displayed in bold in the produced outptut.
Help users to identify which is the like that must be used as reference for deletion.